tools/obscuroscan/obscuroscan.go

```diff
@@ -83,26 +87,32 @@ func NewObscuroscan(address string) *Obscuroscan {
 func (o *Obscuroscan) Serve(hostAndPort string) {
 	serveMux := http.NewServeMux()
 
-	serveMux.HandleFunc(pathNumRollups, o.getNumRollups)            // Get the number of published rollups.
-	serveMux.HandleFunc(pathNumTxs, o.getNumTransactions)           // Get the number of rolled-up transactions.
-	serveMux.HandleFunc(pathGetRollupTime, o.getRollupTime)         // Get the average rollup time.
-	serveMux.HandleFunc(pathLatestRollups, o.getLatestRollups)      // Get the latest rollup numbers.
-	serveMux.HandleFunc(pathLatestTxs, o.getLatestTxs)              // Get the latest transaction hashes.
-	serveMux.HandleFunc(pathRollup, o.getRollupByNumOrTxHash)       // Get the rollup given its number or the hash of a transaction it contains.
-	serveMux.HandleFunc(pathBlock, o.getBlock)                      // Get the L1 block with the given number.
-	serveMux.HandleFunc(pathDecryptTxBlob, o.decryptTxBlob)         // Decrypt a transaction blob.
-	serveMux.HandleFunc(pathAttestation, o.attestation)             // Retrieve the node's attestation.
-	serveMux.HandleFunc(pathAttestationReport, o.attestationReport) // Retrieve the node's attestation report.
+	serveMux.HandleFunc(pathAPI+pathNumRollups, o.getNumRollups)            // Get the number of published rollups.
```

Comment:
Without this prefix, if you requested `/attestation`, it thought you meant the endpoint, not the page.